src/main/kotlin/traffic_simulation/Main.kt

```diff
@@ -0,0 +1,15 @@
+package traffic_simulation
+
+fun main(args: Array<String>) {
+
+    scenario(testList, testRoad)
```

Comment:
Es ist typischerweise sinnvoll, "Dinge" (was auch immer "Dinge" im konkreten Kontext genau heißt) dort zu erstellen, wo man sie auch braucht. In diesem Fall wäre es deswegen besser `testList` und `testRoad` in der `scneario` Methode zu erstellen.